app/page.js

```diff
@@ -5,10 +5,13 @@ import React, { useCallback, useContext, useEffect, useState } from 'react';
 import axios from '@/utils/api';
 import { useRouter } from 'next/navigation';
 
+// FOR LAYOUTS
+import Navbar from '@/components/navbar';
+
```

Comment:
I think imports should be put together and FOR LAYOUTS seem like unnecessary comment